{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/svg/fa-th.svg

```diff
@@ -0,0 +1 @@
+<svg class="svg-Image" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M512 1248v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm0-512v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm640 512v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm-640-1024v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm640 512v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm640 512v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm-640-1024v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm640 512v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68zm0-512v192q0 40-28 68t-68 28h-320q-40 0-68-28t-28-68v-192q0-40 28-68t68-28h320q40 0 68 28t28 68z"/></svg>
```

Comment:
Is the class used for anything? Also, the filename for this SVG might be nice if it was descriptive i.e. grid.svg

Reply:
The class sizes the SVG in CSS.

`fa-th` was to show that it comes from Font Awesome - same as the other ones - (this would be its name in HTML if we were using FA) but I can change that.

Reply:
Fair enough didn't spot that, I guess if we have other SVG's named in the same format as well then I guess it makes sense for them all to be consistent, but not sure we really need to know where it came from personally I'd just prefer a name that describes what it is

Reply:
...but on second thoughts, you're totally right, I've re-organised these.